doc/specs/stdlib_hashmaps.md

```diff
@@ -287,8 +287,8 @@ is an `intent(out)` argument.
       end do
       allocate(other_in % value, source=dummy_val)
       call copy_other( other_in, other_out )
-      select type(other_out)
-	  type(dummy_type)
+      select type(other_out)   !there is an issue here, please check.
```

Comment:
What is the issue here?

Reply:
I don't know if it is an issue with the code or the compiler. gfortran11 doesn't want to compile it.

Reply:
The issue is the next line, no? `typeis` should be `type is` with a whitespace. Both nagfor and Intel also complain about this, gfortran 7.5 was sloppy about it (probably also 9.x), but gfortran 11 caught it.

Reply:
Actually, it must be something different, gfortran also doesn't complain about `typeis` (vs. `type is`) in general.

Reply:
I just fixed the issue. The problem was that the check must be on `other_out %value` instead of `other_out`. Thank for your input.